lib/theme.dart

```diff
@@ -0,0 +1,43 @@
+/**
+ * Custom theme
+ */
+import 'package:flutter/material.dart';
+
+final ThemeData randomRunThemeData = ThemeData(
+  primaryColor: RandomRunColors.brightPink,
+  fontFamily: 'Montserrat'
+);
+
+final Text appBar = Text(
+  'Random Run',
+  style: TextStyle(
+    fontFamily: 'Montserrat',
+    fontSize: FontSize.small,
+    fontWeight: FontWeight.w600
+  )
+);
+
+class RandomRunColors {
+  RandomRunColors._();
```

Comment:
whats this do?

Reply:
so I was following this: https://gist.github.com/mikemimik/5ac2fa98fe6d132098603c1bd40263d5
but I just tried removing it and it works, so I don't think we have to do that (maybe it was an old version of flutter that required it to instantiate a class)